docs/content/docs/tasks_usage/namespace_monitoring.md

```diff
@@ -0,0 +1,148 @@
+Enrolling a namespace also optionally allows for metrics to be collected for resources in the given namespace and for Pods in the namespace to be automatically injected with sidecar proxy containers. These are all features that help OSM provide functionality for traffic management and observability. Scoping this functionality at the namespace level allows teams to organize which segments of
+their cluster should be part of which mesh.
+
+Namespace monitoring, automatic sidecar injection, and metrics collection is controlled by adding certain labels and annotations to a Kubernetes namespace. This can be done manually or using the `osm` CLI although using the `osm` CLI is the recommended approach. The presence of the label `openservicemesh.io/monitored-by=<mesh-name>` allows an OSM control place with the given `mesh-name` to monitor
+all resources within that namespace. The annotation `openservicemesh.io/sidecar-injection=enabled` enables OSM to automatically inject sidecar proxy containers in all Pods created within that namespace. The metrics annotation `openservicemesh.io/metrics=enabled` allows OSM to collect metrics on resources within a Namespace.
+
+See how to use the OSM CLI to manage namespace monitoring below.
+
+## Adding a Namespace to the OSM Control Plane
+
+Add a namespace for monitoring and sidecar injection to the mesh with the following command:
+
+```bash
+osm namespace add <mesh-name>
+```
+
+Explicitly disable sidecar injection while adding the namespace using `--disable-sidecar-injection flag as show [here](../tasks_usage/sidecar_injection.md#Explicitly-Disabling-Automatic-Sidecar-Injection-on-Namespaces).
+
+## Remove a Namespace from the OSM control plane
+
+Remove a namespace from being monitored by the mesh and disable sidecar injection with the following command:
+
+```bash
+osm namespace remove <mesh-name>
+```
+
+This command will remove the OSM specific labels and annotations on the namespace thus removing it from the mesh.
+
+## Enable Metrics for a Namespace
+
+```bash
+osm metrics enable --namespace <namespace>
+```
+
+## Ignore a Namespace
+
+There may be namespaces in a cluster that should never be part of a mesh. To explicity exclude a namespace from OSM:
+
+``` bash
+osm namespace ignore <namespace>
+```
+
+## List Namespaces Part of a Mesh
+
+To list namespaces within a specific mesh:
+
+```bash
+osm namespace list --mesh-name=<mesh-name>
+```
+
+## Troubleshooting Guide
+
+### Policy Issues
+
+If you're not seeing changes in SMI policies being applied to a resources in a namespace, ensure the namespace is enrolled in the correct mesh:
+
+```bash
+osm namespace list --mesh-name=<mesh-name>
+NAMESPACE         MESH   SIDECAR-INJECTION
+<namespace>       osm    enabled
+```
+
+If the namespace does not show up, check the labels on the namespace using `kubectl`:
+
+```bash
+$ kubectl get namespace <namespace> --show-labels
```

Comment:
nit for consistency: `$` in some commands but not all

Reply:
resolved. thanks